### ISSUE TITLE
Please update -  Plugin dont work with DokuWiki Release 2023-04-04a "Jack Jackrum"

### ISSUE THREAD

**Issue by TBobsin** [GreenItSolutions/dokuwiki-plugin-switchpanel#32]
Hello
the plug-in don't work with newer version of dokuwiki.
i am using Release 2023-04-04a "Jack Jackrum" 

there are no errors in the logfiles. 

the graphic wont be rendered. see screenshot. 
![Screenshot 2023-06-16 002317](https://github.com/GreenItSolutions/dokuwiki-plugin-switchpanel/assets/50228256/b40fc007-6acf-4d32-bf90-5205cda3fb83)

**Comment by lloydbayley** [comment]
I don't think this is being maintained anymore, If you're keen to fix this, the answer is in #29 
You need to edit the syntax.php
Worked for me! :)
